std_msgs/UInt16 req_id
std_msgs/UInt8 status
std_msgs/String text 
